pkg/component/controller/kine.go

```diff
@@ -110,3 +122,25 @@ func (k *Kine) Start(_ context.Context) error {
 func (k *Kine) Stop() error {
 	return k.supervisor.Stop()
 }
+
+const hcKey = "/k0s-health-check"
+const hcValue = "value"
+
+func (k *Kine) Healthy() error {
+	ok, err := k.bypassClient.Write(k.ctx, hcKey, hcValue, 64)
```

Comment:
What about using a value that's (reasonably) unique value for each call to `Healthy`? Say, the current timestamp.

Reply:
I think that makes sense.

Reply:
Given the current logic, I think we should either do it for the key, or not at all, since the Write method doesn't overwrite already existing values. That would make the health test fail later on when it reads a previously written value that didn't expire yet.